packages/landing/src/components/layer.switcher.dropdown.tsx

```diff
@@ -43,47 +55,44 @@ export class LayerSwitcherDropdown extends Component<unknown, LayerSwitcherDropd
     window.history.pushState(null, '', `?${MapConfig.toUrl(Config.map)}`);
   };
 
-  onClick = (e: Event): void => {
+  onZoomExtentChange: ChangeEventHandler<unknown> = (e) => {
     const target = e.target as HTMLInputElement;
     this.setState({ zoomToExtent: target.checked });
   };
 
-  render(): ComponentChild {
+  render(): ReactNode {
+    const ret = this.makeOptions();
+
     return (
-      <div class="LuiDeprecatedForms">
+      <div className="LuiDeprecatedForms">
```

Comment:
Maybe we should stop using these `LuiDeprecatedForms` later as already upgraded to latest lui.

Reply:
I was trying to avoid making too many changes, could be lots of other improvements we could do here.